components/merkledb/Cargo.toml

```diff
@@ -66,5 +66,8 @@ rocksdb_bzip2 = ["rocksdb/bzip2"]
 [[example]]
 name = "blockchain"
 
+[[example]]
```

Comment:
Hm, do examples really need to be listed here? I'd rather remove the mention of the `blockchain` example if this doesn't break anything.